build_config/lib/src/build_config.dart

```diff
@@ -124,19 +127,21 @@ class BuildConfig {
   }
 
   BuildConfig({
-    String packageName,
-    @required Map<String, BuildTarget> buildTargets,
-    Map<String, GlobalBuilderConfig> globalOptions,
-    Map<String, BuilderDefinition> builderDefinitions,
-    Map<String, PostProcessBuilderDefinition> postProcessBuilderDefinitions =
+    String? packageName,
```

Comment:
I'm surprised we allows a missing package name - I guess since this wasn't `@required` before we have some usages with no name?

Reply:
I can't actually find usages of this, I will try just making it required.

Reply:
Ah, its because the generated code uses it.